client/src/components/LoginContainer.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Route, NavLink } from 'react-router-dom';
+import RegisterLogin from './Register';
+import Users from './Users';
+
+const LoginWrapper = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 40%;
+  min-height: 400px;
+  border: 3px solid ${props => props.theme.gray};
+  border-radius: 8px;
+  background-color: ${props => props.theme.dBlue}
+  margin: 40px auto;
+  color: ${props => props.theme.red}
+  box-shadow: 4px 4px 22px ${props => props.theme.lBlue}
+`;
+
+const LoginNav = styled.nav`
+  width: 100%;
+  height: 50px;
+  background-color: ${props => props.theme.red}
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  color: black;
+
+  a {
+    text-decoration: none;
+    font-size: 20px;
+    color: ${props => props.theme.gray};
+    opacity: .7;
+
+  }
+
+  .active {
+    font-weight: bold;
+    opacity: 1;
+  }
+`;
+
+class LoginContainer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      loggedIn: false,
+    };
+  }
+
+  logInUser = token => {
+    this.setState({ loggedIn: true });
+    localStorage.setItem('jwt', token);
+  };
+
+  logOutUser = () => {
+    this.setState({ loggedIn: false });
+    localStorage.removeItem('jwt');
```

Comment:
I think your git commits are saying you got this all done yesterday. Getting the local storage working w/ the jwt (as well as the styled-components, rest of front-end, back-end) this is a crazy amount of work to pull off. Great job.